src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/codeSmell/ParameterEqualsDefaultValueInspector.java

```diff
@@ -0,0 +1,140 @@
+package com.kalessil.phpStorm.phpInspectionsEA.inspectors.codeSmell;
+
+import com.intellij.codeInspection.LocalQuickFix;
+import com.intellij.codeInspection.ProblemDescriptor;
+import com.intellij.codeInspection.ProblemHighlightType;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.lang.ASTNode;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiElementVisitor;
+import com.intellij.psi.SmartPointerManager;
+import com.intellij.psi.SmartPsiElementPointer;
+import com.jetbrains.php.lang.psi.elements.*;
+import com.kalessil.phpStorm.phpInspectionsEA.openApi.BasePhpElementVisitor;
+import com.kalessil.phpStorm.phpInspectionsEA.openApi.BasePhpInspection;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This file is part of the Php Inspections (EA Extended) package.
+ *
+ * (c) Vladimir Reznichenko <kalessil@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+public class ParameterEqualsDefaultValueInspector extends BasePhpInspection {
+    private static final String message = "This parameter could be dropped, because the value is the same from default value.";
```

Comment:
I think we should change word `parameter` to the `argument`
There is big difference: http://stackoverflow.com/questions/156767/whats-the-difference-between-an-argument-and-a-parameter
Maybe this message will be better: `Argument can be safely dropped. Default value is the same `